packages/client-proxy/main.go

```diff
@@ -213,6 +227,13 @@ func main() {
 		logger.Info("waiting 15 seconds before shutting down http service")
 		time.Sleep(15 * time.Second)
 
+		logger.Info("shutting down telemetry")
+
+		err := cleanupTelemetry(ctx)
```

Comment:
cleanupTelemetry is called twice, also it should be called after the server shut downs, that was the whole point of this

Reply:
i should be able to just remove this then right? the defer statement above should execute after the go routines stop which is after the http service is completed

Reply:
[https://github.com/e2b-dev/infra/pull/364](https://github.com/e2b-dev/infra/pull/364)